src/app/static/src/js/components/runWorkflow/runWorkflowTabs.vue

```diff
@@ -11,11 +11,21 @@
                         <div class="d-md-block mt-4 mt-md-0 collapse navbar-collapse" id="sidebar">
                             <ul class="nav flex-column list-unstyled mb-0">
                                 <li class="nav-item">
-                                    <a id="run-workflow-link" class="nav-link active" data-toggle="tab" role="tab" href="#"
+                                    <a id="run-workflow-link"
+                                       class="nav-link"
+                                       :class="selectedTab === 'runWorkflow' ? 'active' : ''"
```

Comment:
An alternative, but simply a matter of taste:
```suggestion
                                       :class="{ active: selectedTab === 'runWorkflow' }"
```